Tridibmodified

Tridib11 opened the pull request:
| Fixes       | # Issue No. |
| ----        |    :----:   |    

# Description

<!--Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.-->

## Type of change

<!--Please delete options that are not relevant.-->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New project (non-breaking change which adds functionality)

# How Has This Been Tested?

<!--Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration-->

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have included a README.md file
- [ ] My project follows this structure: Domain Folder/ Project Name Folder/ Project files and Readme.md files
- [ ] I have added demonstration in the form of GIF/video file
- [ ] I am a GSSoC'22 Participant

manisha-27 reviewed (changes requested):
Hey @Tridib11 I'm unable to figure out, so please explain me in which folder you added your project also where is your readme file as per #43 and video as per #178.
If by any chance you are unable to understand how to arrange all your files in the domain folder or need any help, Just ping me.

Tridib11 commented:
Hey actually what just happend everything was going perfectly fine but the
the last pull reuqest i made i dont understand my everything was ready the
project the vedio and as soon as i am trying to make a pull request half of
my files are getting deleted and only the xml files are staying i tried it
again and again but its showing the same thing i dont understand whats the
problem. Should i send you my project in personal?

On Mon, 21 Mar 2022, 17:37 Manisha Singh, ***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> Hey @Tridib11 <https://github.com/Tridib11> I'm unable to figure out, so
> please explain me in which folder you added your project also where is your
> readme file as per #43
> <https://github.com/abhijeet007rocks8/Dev-Scripts/issues/43> and video as
> per #178 <https://github.com/abhijeet007rocks8/Dev-Scripts/issues/178>.
> If by any chance you are unable to understand how to arrange all your
> files in the domain folder or need any help, Just ping me.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/abhijeet007rocks8/Dev-Scripts/pull/194#pullrequestreview-915635494>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AURRZHULV6C2AMBYYLHIPJTVBBQ63ANCNFSM5RELQJHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

manisha-27 commented:
Okay, No worries.
I suppose you already have your project on your local machine?
And, I think you actually created 3 branches, and after that I couldn't understand, maybe you were pulling request from other other branches considering the same project.
Check if this is the issue?

Tridib11 commented:
Yes i have the project on my local machine,i tried to fork the repo again
added the files in the java folder then made a new branch and tried to make
a pr its having the same problem only the .xml file stays the rest gets
deleted.

Should i send you my project in personal if its not a problem.?

On Mon, 21 Mar 2022, 22:42 Manisha Singh, ***@***.***> wrote:

> Okay, No worries.
> I suppose you already have your project on your local machine?
> And, I think you actually created 3 branches, and after that I couldn't
> understand, maybe you were pulling request from other other branches
> considering the same project.
> Check if this is the issue?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/abhijeet007rocks8/Dev-Scripts/pull/194#issuecomment-1074174915>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AURRZHU2W66VICLAQRFQ7YDVBCUWHANCNFSM5RELQJHA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

manisha-27 commented:
Okay, just ping me in the devscript channel of discord, will see there.
